### PR TITLE
update to latest version of clone factory contract

### PR DESCRIPTION
**Pull request by lsaether**

### CODE REVIEW

**Comment by kosecki123** [comment]
What's the "features" in this release?

**Comment by lsaether** [comment]
no additional core features, just updating to the newer version of the bytecode (more audited + optimized)

There is a proxy contract probe that is deploy for mainnet which would allow for detecting these contracts and there is a push to include the delegated contract as the "source" contract on interfaces like etherscan. more info https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1167.md

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     },
     "homepage": "https://github.com/chronologic/ethereum-alarm-clock#readme",
     "dependencies": {
-        "@optionality.io/clone-factory": "^0.1.2",
+        "@optionality.io/clone-factory": "0.1.6",
```
Comment (kosecki123) [inline package.json]: Is there are reason we are keeping this npm package?

Comment (lsaether) [reply package.json]: For reference, I just copied it from there.

Comment (lsaether) [reply package.json]:
```

**Pull request merged**